frontend/stylesheets/_utils.less

```diff
@@ -66,6 +66,18 @@ td.nowrap * {
   float: right !important;
 }
 
+.clearfix::before {
```

Comment:
I believe these `clearfix` rules were previously provided by some antd default styles/resets. I took these CSS rules directly from the "old" CSS file that we shipped with antd4.

Reply:
Bootstrap has a better explanation then me: https://getbootstrap.com/docs/4.0/utilities/clearfix/